src/utils/parseAttributes.ts

```diff
@@ -41,6 +41,7 @@ export const parseProjectAttributes = (
       from: projectAttribute.from,
       to: projectAttribute.to,
       name: attribute.name,
+      type: type,
```

Comment:
this function could be replaced by 

```
export const parseProjectAttributes = (
  attributes: ProjectAttributeResponse[] | undefined,
): ParsedProjectAttributedMap => {
  if (!attributes?.length) return {};
  const attributesByType: ParsedProjectAttributedMap = {};
  attributes?.forEach((projectAttribute) => {
    const { attribute } = projectAttribute;
    pushParsedProjectAttribute(attribute, attributesByType);
  });
  return attributesByType;
};
```

So re can reutilize `pushParsedProjectAttribute`